.github/workflows/ci_cd.yml

```diff
@@ -404,6 +404,9 @@ jobs:
           name: api_logs
           path: api_logs
 
+      - name: Run API schema tests
+        run: just api/test-schema
```

Comment:
This would fit better as a matrix entry in the `django-checks` job below, as those are all checks based on `just` recipes. Additionally the recipe this depends on i.e. `doc-test` is also a job in that matrix.